File: board_dsl/src/main/java/com/dooho/board/dto/user/MyPageDto.java
package com.dooho.board.dto.user;

import com.dooho.board.entity.BoardEntity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MyPageDto {
    private String userEmail;
    private String userNickname;
    private String userProfile;
    private List<BoardEntity> userBoard;

}

File: board_spring/src/main/java/com/dooho/board/dto/user/MyPageDto.kt
package com.dooho.board.dto.user

import com.dooho.board.entity.BoardEntity


data class MyPageDto (
    val userEmail: String? = null,
    val userNickname: String? = null,
    val userProfile: String? = null,
    val userBoard: List<BoardEntity?>? = null
    )